src/controllers/login.js

```diff
@@ -0,0 +1,29 @@
+const prisma = require("../utils/prisma.js");
+const jwt = require("jsonwebtoken");
+
+
+const { WrongPasswordError, CantFindIdError } = require("../utils/errors");
+const { } = require("../utils/prisma.js");
+
+const login = async (req, res) => {
+    const loginData = req.body
+    console.log(loginData)
+    const user = await prisma.user.findUniqueOrThrow({
+        where: {userName: loginData.username}
+    })
+    if (!user) {
+        throw new CantFindIdError()
+    }
+    if (user.password !== loginData.password) {
+        throw new WrongPasswordError()
+    }
+    let token = jwt.sign(user.id, process.env.JWT_SECRET_KEY);
```

Comment:
The payload should be a JSON object rather than a primitive type, `{ id: user.id }`. This allows things to be added to the payload when needed, such as an `iat` (issued at time stamp) or a username if it becomes desirable

Reply:
```js
let token = jwt.sign({id: user.id}, process.env.JWT_SECRET_KEY);
```

Fixed!